crates/red_knot/src/symbols.rs

```diff
@@ -244,9 +244,8 @@ impl SymbolTable {
         let root_scope_id = SymbolTable::root_scope_id();
         let mut builder = SymbolTableBuilder {
             table: SymbolTable::new(),
-            scopes: vec![root_scope_id],
+            scopes: vec![(root_scope_id, FlowGraph::start())],
```

Comment:
Nit: I think I would prefer a named struct here to awoid the awkward `.1` and `.0` which I always find hard to read.